hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -468,6 +458,17 @@ private synchronized void addEntryToDnsToUuidMap(
     dnList.add(uuid);
   }
 
+  @SuppressFBWarnings(value = "AT_OPERATION_SEQUENCE_ON_CONCURRENT_ABSTRACTION")
+  private synchronized void addEntryToHostNmToUuidMap(
+      String hostName, String uuid) {
+    Set<String> dnList = hostNmToUuidMap.get(hostName);
+    if (dnList == null) {
+      dnList = ConcurrentHashMap.newKeySet();
+      hostNmToUuidMap.put(hostName, dnList);
+    }
+    dnList.add(uuid);
```

Comment:
We can simplify the code and avoid having to add a suppression:

```suggestion
  private void addEntryToHostNameToUuidMap(
      String hostName, String uuid) {
    hostNmToUuidMap
        .computeIfAbsent(hostName, any -> ConcurrentHashMap.newKeySet())
        .add(uuid);
```